ui/raidboss/raidemulator.js

```diff
@@ -18,6 +18,12 @@ import { TimelineLoader } from './timeline';
 import Tooltip from './emulator/ui/Tooltip';
 import UserConfig from '../../resources/user_config';
 import raidbossFileData from './data/manifest.txt';
+// @TODO: For some reason this absolutely has to be imported as `Worker`, no matter what settings
```

Comment:
Maybe this, if you want to name the constructor?
```
        {
          // Worker has to go before normal js
          test: /NetworkLogConverterWorker\.(?:c|m)?js$/,
          loader: 'worker-loader',
          options: {
            esModule: true,
            inline: 'fallback',
            worker: {
              type: 'SharedWorker',
              options: {
                type: 'classic',
                name: 'NetworkLogConverterWorker',
              },
            },
          },
          resolve: {
            fullySpecified: false,
          },
        },
```

Reply:
A slightly modified version of this (`Worker` instead of `SharedWorker`) works. Fix pushed.